pkg/controllers/raycluster_webhook.go

```diff
@@ -161,8 +161,8 @@ func (w *rayClusterWebhook) ValidateUpdate(ctx context.Context, oldObj, newObj r
 
 	// Init Container related errors
 	if ptr.Deref(w.Config.MTLSEnabled, true) {
-		allErrors = append(allErrors, validateHeadInitContainer(rayCluster, w.Config.IngressDomain)...)
-		allErrors = append(allErrors, validateWorkerInitContainer(rayCluster)...)
+		allErrors = append(allErrors, validateHeadInitContainer(rayCluster, w.Config.IngressDomain, w.Config.CertGeneratorImage)...)
+		allErrors = append(allErrors, validateWorkerInitContainer(rayCluster, w.Config.CertGeneratorImage)...)
```

Comment:
Can we change these to either take `w` or `w.Config` directly that way if in the future other parts of the struct are needed they can be dereferenced directly in the function rather than adding more and more args.

https://refactoring.guru/preserve-whole-object

Reply:
Updated to take w.config instead. Thank you.